Extensions/PathfindingBehavior/pathfindingruntimebehavior.ts

```diff
@@ -756,12 +923,44 @@ namespace gdjs {
         return newNode;
       }
 
+      _moveStampHitBoxesTo(x: float, y: float) {
+        for (let pi = 0; pi < this._objectHitboxes.length; ++pi) {
+          const objectPolygon = this._objectHitboxes[pi];
+          const stampPolygon = this._stampHitboxes[pi];
+          for (let vi = 0; vi < objectPolygon.vertices.length; ++vi) {
+            const objectPoint = objectPolygon.vertices[vi];
+            const stampPoint = stampPolygon.vertices[vi];
+            stampPoint[0] = objectPoint[0] + x;
+            stampPoint[1] = objectPoint[1] + y;
+          }
+        }
+      }
+
+      _checkCollision(hitboxes: Polygon[]): boolean {
```

Comment:
The method name should I think indicate that this is checking collision for the stamp hitboxes